### ISSUE TITLE
Disable homepage, demo and sandbox

### ISSUE THREAD

**Issue by dmarcoux** [almarklein/timetagger#511]
Hello,

first of all, thank you for TimeTagger. I am self-hosting it with Docker for my personal use and I wonder if it would be possible to disable the homepage, demo and sandbox pages. Since I am the sole user of my TimeTagger instance, it doesn't really make sense for me to have all of this. Disabling those pages and removing their navigation links on the account page would be awesome. Maybe this can be done with environment variables?

**Comment by Tagirijus** [comment]
I know that such answers might not be the thing one is looking for, hehe, but: why not just bookmark the link directly to the tracking app so that you do not have to care about the other pages at all? I do this all the time.

**Comment by almarklein** [comment]
Yeah, I don't expect people to spend much tome on that landing page, and only use the page for the app. If you really want to, you can easily roll your own boot script (copy [__main__.py](https://github.com/almarklein/timetagger/blob/main/timetagger/__main__.py) and make your changes).
